herddb-core/src/test/java/herddb/core/SimpleOperatorsTest.java

```diff
@@ -287,6 +287,16 @@ public void simpleArithmeticOperationsTest() throws Exception {
             try (DataScanner scan1 = scan(manager, "SELECT * FROM tblspace1.tsql WHERE 'AbBbCc' LIKE '%AbBbCc%'", Collections.emptyList());) {
                 assertEquals(1, scan1.consume().size());
             }
+            
+            // bad "constant" pattern
+            herddb.utils.TestUtils.assertThrows(herddb.core.HerdDBInternalException.class, () -> {
+                scan(manager, "SELECT * FROM tblspace1.tsql WHERE 'AbBbCc' LIKE '%AbBbC(.c%'", Collections.emptyList());
```

Comment:
this part is to be dropped @diegosalvi 
it would be awesome to add a test with a wrong pattern, but I guess there is no real "bad" patten for LIKE sintax